app/helpers/authentication_helper.rb

```diff
@@ -27,10 +27,10 @@ def signed_up_with(user = current_user)
     providers = Authentication::Providers.enabled_for_user(user)
 
     # If the user did not authenticate with any provider, they signed up with an email.
-    auth_method = providers.any? ? providers.map(&:official_name).to_sentence : "Email & Password"
-    verb = providers.size > 1 ? "any of those" : "that"
+    auth_method = providers.any? ? providers.map(&:official_name).to_sentence : I18n.t("helpers.authentication_helper.email_password") # rubocop:disable Layout/LineLength
```

Comment:
I don't think the `rubocop:disable` should go to the end of the line here on 2 consecutive lines, please use this form instead:

```ruby
# rubocop:disable Layout/LineLength
auth_method = ...
verb = ...
# rubocop:enable Layout/LineLength
```